### ISSUE TITLE
social.network.europa.eu doesn't always work

### ISSUE THREAD

**Issue by HolzmanoLagrene** [WebBreacher/WhatsMyName#677]
Even though the proposed link for `social.network.europa.eu` works for some usernames, I had the case where the page would just load forever. 

![image](https://user-images.githubusercontent.com/23474309/217606355-48bde4e7-9bcb-425f-ad82-6721276eaa5b.png)

I checked that in the background the request is made via xhr to the normal API and returned a proper json response object.

Request:
`https://social.network.europa.eu/api/v1/accounts/lookup?acct=non_existing.`
Result:
`{"error":"Record not found"}`

I would therefore suggest to change [`wmn-data.json`](https://github.com/WebBreacher/WhatsMyName/blob/main/wmn-data.json) from

```json
{
  "name" : "Mastodon-EU_Voice",
  "uri_check" : "https://social.network.europa.eu/@{account}",
  "post_body" : "",
  "e_code" : 200,
  "e_string" : "social.network.europa.eu",
  "m_string" : "The page you are looking for isn&#39;t here.",
  "m_code" : 404,
  "known" : ["EC_DIGIT", "EUSPA"],
  "cat" : "social",
  "valid" : true
}
```

to

```json
 {
  "name" : "Mastodon-EU_Voice",
  "uri_check" : "https://social.network.europa.eu/api/v1/accounts/lookup?acct={account}",
  "post_body" : "",
  "e_code" : 200,
  "e_string" : "social.network.europa.eu",
  "m_string" :  "Record not found",
  "m_code" : 404,
  "known" : ["EC_DIGIT", "EUSPA"],
  "cat" : "social",
  "valid" : true
}
```

**Comment by WebBreacher** [comment]
I've checked and this makes sense. I would like it if you moved the current `uri_check` to the `uri_pretty` value (`"uri_pretty" : "https://social.network.europa.eu/@{account}",`

@HolzmanoLagrene are you able to/willing to make this change and Pull Request or do you need us to do it?

**Comment by WebBreacher** [comment]
This was fixed in #682
